pkg/gokudos/kudos_service.go

```diff
@@ -0,0 +1,15 @@
+package gokudos
+
+import (
+	"github.com/slack-go/slack"
+	"net/http"
+)
+
+type KudosService interface {
```

Comment:
Maybe it would be a good idea to keep http and service layer apart? So lets say 
AddKudos would accept something like
`AddKudos(ctx context.Context, kudos Kudos)error`
and it would validate the kudos and ask the storage to persist it,
and then http layer would only have to unwrap slack command from HTTP and ask service to AddKudos?

WDYT?